Bio/Seq.py

```diff
@@ -75,7 +75,7 @@ class Seq:
 
     The Seq object also provides some biological methods, such as complement,
     reverse_complement, transcribe, back_transcribe and translate (which are
-    not applicable to sequences with a protein alphabet).
+    not applicable to protein sequences).
     """
 
     def __init__(self, data, alphabet=Alphabet.generic_alphabet):
```

Comment:
The alphabet here will be removed in a separate PR?

Reply:
Yes the (removed) deprecation warning version showed we still use it but we're getting close.